far/string_utils.cpp

```diff
@@ -399,11 +412,16 @@ TEST_CASE("string.case")
 
 TEST_CASE("string.utils")
 {
-	for (const auto& i: GetSpaces())
+	for (const auto& i: GetBlanks())
```

Comment:
I think this whole test case is redundant - everything in it is already checked in string.blanks, string.spaces, string.eols above. Could you remove the whole thing please?

Reply:
Done. I pulled `isblank` test up into `"string.blanks"` though, just in case.